cypress-tests/cypress/e2e/PaymentUtils/Novalnet.js

```diff
@@ -1,193 +1,182 @@
-        },
+    },
+    "3DSManualCapture": {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulThreeDSTestCardDetails,
+        },
+        customer_acceptance: null,
+        setup_future_usage: "on_session",
       },
-      No3DSManualCapture: {
-        Request: {
-          payment_method: "card",
-          payment_method_data: {
-            card: successfulNo3DSCardDetails,
-          },
-          currency: "EUR",
-          customer_acceptance: null,
-          setup_future_usage: "on_session",
-        },
-        Response: {
-          status: 200,
-          body: {
-            status: "requires_capture",
-          },
+      Response: {
+        status: 200,
+        body: {
+          status: "requires_capture",
         },
       },
-      No3DSAutoCapture: {
-        Request: {
-          payment_method: "card",
-          payment_method_data: {
-            card: successfulNo3DSCardDetails,
-          },
-          currency: "EUR",
-          customer_acceptance: null,
-          setup_future_usage: "on_session",
-        },
-        Response: {
-          status: 200,
-          body: {
-            status: "succeeded",
-          },
-        },
+    },
+    "3DSAutoCapture": {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulThreeDSTestCardDetails,
+        },
+        customer_acceptance: null,
+        setup_future_usage: "on_session",
       },
-      Capture: {
-        Request: {
-          payment_method: "card",
-          payment_method_data: {
-            card: successfulNo3DSCardDetails,
-          },
-          currency: "EUR",
-          customer_acceptance: null,
+      Response: {
+        status: 200,
+        body: {
+          status: "requires_customer_action",
         },
-        Response: {
-          status: 200,
-          body: {
-            status: "succeeded",
-            amount: 6500,
-            amount_capturable: 0,
-            amount_received: 6500,
-          },
+      },
+    },
+    //TODO: Add No3DSManualCapture, No3DSAutoCapture
+    // No3DSManualCapture: {
+    //   Request: {
+    //     payment_method: "card",
+    //     payment_method_data: {
+    //       card: successfulNo3DSCardDetails,
+    //     },
+    //     customer_acceptance: null,
+    //     setup_future_usage: "on_session",
+    //   },
+    //   Response: {
+    //     status: 200,
+    //     body: {
+    //       status: "requires_capture",
+    //     },
+    //   },
+    // },
+    // No3DSAutoCapture: {
+    //   Request: {
+    //     payment_method: "card",
+    //     payment_method_data: {
+    //       card: successfulNo3DSCardDetails,
+    //     },
+    //     customer_acceptance: null,
+    //     setup_future_usage: "on_session",
+    //   },
+    //   Response: {
+    //     status: 200,
+    //     body: {
+    //       status: "succeeded",
+    //     },
+    //   },
+    // },
```

Comment:
can be removed.

Reply:
@pixincreate connector supports this, I am facing some issues in the non3ds flow, once that is resolved, this should work

Reply:
CC: @Gnanasundari24